lib/actions/actionsBuilder.js

```diff
@@ -0,0 +1,30 @@
+const AddLinkToProjectAction = require('./AddLinkToProjectAction');
+const assert = require('assert');
+
+class ActionsBuilder {
+  constructor(){
+    this.commandMap = {
+      'add': new AddLinkToProjectAction,
+    }
+  }
+
+  parseActionTerm(userInputString = '') {
```

Comment:
Why a default? Is it not required?

Reply:
I set this in correlation with these lines
let parsedUserInput = userInputString.toLowerCase().split(' ');
if (parsedUserInput.length < 3){
this forces a defined input to test.. and so going forward if parsedUserInput === 0 its an invalid input

is there another way to achieve this?

Reply:
By adding a default, you're telling the caller that you don't have to provide a value and things will be ok. I recommend you not provide a default value for this parameter.